src/css/controls/flags/audioplayer.less

```diff
@@ -57,9 +61,66 @@
         }
 
         .jw-slider-time {
-            display: flex;
             flex: 0 1 auto;
             align-items: center;
+            display: flex;
+            order: 1;
+        }
+
+        .jw-icon-volume {
+            margin-right: 0;
+            transition: margin-right @default-transition;
+
+            .jw-overlay {
+                display: none;
+            }
+        }
+
+        .jw-horizontal-volume-container {
+            transition: width 300ms @default-timing-function;
```

Comment:
Can we use `transform` with `scaleX(0.0-1.0)` instead of `width`?

Reply:
This pushes the other elements when the horizontal volume is not open.